### DISCUSSION TITLE
Visual Designs for CAD Sketcher

### DISCUSSION THREAD

**Discussion by shreeneet** [hlorus/CAD_Sketcher#449] [Feature Requests]
Hello team,
Felt super connected reading, "facilitating precision modeling within blender".
The usability of the app, outlining features as well as prioritizing features for development are better done with structured frameworks. & I would like to contribute my share with these aspects along with varying levels of involvements on Interaction designs, Visual Designs, Icons, Website etc of CAD Sketcher.

**Comment by hlorus** [comment]
Thanks for your support! I'd suggest reaching out to other contributors on [discord](https://discord.com/channels/882152824249667594/967316874121588777) or directly commenting on github issues.
